src/test/java/seedu/address/model/person/NameContainsKeywordsPredicateTest.java

```diff
@@ -46,19 +46,19 @@ public void test_nameContainsKeywords_returnsTrue() {
         // One keyword
         NameContainsKeywordsPredicateOr predicate =
                 new NameContainsKeywordsPredicateOr(Collections.singletonList("Alice"));
-        assertTrue(predicate.test(new PersonBuilder().withName("Alice Bob").build()));
+        assertFalse(predicate.test(new PersonBuilder().withName("Alice Bob").build()));
```

Comment:
Perhaps just change this to:
`assertTrue(predicate.test(new PersonBuilder().withName("Alice").build()));`